ocs_ci/ocs/acm/acm.py

```diff
@@ -77,6 +76,7 @@ def import_cluster(self, cluster_name, kubeconfig_location):
         ):
             if sample:
                 log.info(f"Cluster: {cluster_name} successfully imported")
+                return True
```

Comment:
if you added  `return True` in the function, could you add `return False` after it? just so the function will return bool argument

Reply:
and please add `Returns: ` section in the docs

Reply:
Good point. The goal is to get out of the loop once the sampler is not None so I'll try `return` first.